### PR TITLE
KudWafter OST

### PR DESCRIPTION
**Pull request by SeraphJACK**

### CODE REVIEW

File: album/KSLA-0174~5.toml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+[album]
+title = "アニメション「クドわふたー」オリジナルサウンドトラック"
+artist = "[Unknown Artist]"
```
Comment (SeraphJACK) [inline album/KSLA-0174~5.toml:3]: TODO

Comment (Yesterday17) [reply album/KSLA-0174~5.toml:3]: Various Artists

File: album/KSLA-0174~5.toml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+title = "Further Away"
+artist = "清水準一"
+
+[[discs]]
+catalog = "KSLA-0175"
+
+[[discs.tracks]]
+title = "Light a Way"
+artist = "鈴湯"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:86]: Unnecessary `normal` type for tracks(disc type implies normal).

File: album/KSLA-0174~5.toml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+[[discs.tracks]]
+title = "Moontan"
+artist = "清水準一"
+type = "absolute"
+
+[[discs.tracks]]
+title = "星守歌"
+artist = "能美クドリャフカ"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:101]: ~~normal~~

File: album/KSLA-0174~5.toml [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+[[discs.tracks]]
+title = "grief"
+artist = "清水準一"
+type = "absolute"
+
+[[discs.tracks]]
+title = "いつか気付くかな"
+artist = "折戸伸治"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:111]: ~~normal~~(Why track type of this artist is normal?)

File: album/KSLA-0174~5.toml [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+[[discs.tracks]]
+title = "Little Melody"
+artist = "麻枝准"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:116]: ~~normal~~ (Artist)

File: album/KSLA-0174~5.toml [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+title = "星屑"
+artist = "霜月はるか"
+type = "normal"
+
+[[discs.tracks]]
+title = "Over The Summer"
+artist = "能美クドリャフカ"
+type = "normal"
+
+
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:148]: ~~New line~~

File: album/KSLA-0174~5.toml [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+[[discs.tracks]]
+title = "Cape of Future"
+artist = "清水準一"
+type = "absolute"
+
+[[discs.tracks]]
+title = "星屑"
+artist = "霜月はるか"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:141]: ~~normal~~

File: album/KSLA-0174~5.toml [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+[[discs.tracks]]
+title = "Over The Summer"
+artist = "能美クドリャフカ"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:146]: ~~normal~~

File: album/KSLA-0174~5.toml [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+
+
+[[discs.tracks]]
+title = "Puppy Dreams"
+artist = "北沢綾香"
+type = "normal"
```
Comment (Yesterday17) [inline album/KSLA-0174~5.toml:152]: ~~normal~~

**Review by Yesterday17 — CHANGES_REQUESTED** [review]
- [ ] Missing category file
- [ ] #16 related KudWafter Discs
  - [ ] [KSLA-0057]『クドわふたー Original SoundTrack』
  - [ ] [KSLA-0066]『クドわふたーアレンジアルバム Albina ~Assorted kudwaf Songs~』
  - [ ] [KSLA-0077]『Kud Wafter Original Sound Track』
  - [ ] [KSLA-0089]『リトルバスターズ！／クドわふたー Piano Arrange Album '''ripresa'''』

**Review by Yesterday17 — APPROVED** [review]

**Pull request merged**